themes/theme-gmd/pages/Cart/components/PaymentBar/components/Content/index.jsx

```diff
@@ -28,7 +31,10 @@ function PaymentBarContent() {
           <Portal name={CART_PAYMENT_BAR_TOTALS_BEFORE} props={totalsPortalProps} />
           <Portal name={CART_PAYMENT_BAR_TOTALS} props={totalsPortalProps}>
             <SubTotal />
+            <Discounts />
             <ShippingCosts />
+            <Tax />
+            <GrandTotal />
```

Comment:
Story AC says, we need implement a config `Cart: Show Cart Total: Yes / No ` to show Grand or not